src/main/kotlin/heist/make/sdk/kt/sse/EventSource.kt

```diff
@@ -0,0 +1,65 @@
+package heist.make.sdk.kt.sse
+
+import heist.make.sdk.kt.event.EventTarget
+import java.io.BufferedReader
+import java.io.InputStream
+import java.io.InputStreamReader
+import java.net.HttpURLConnection
+import java.net.URI
+import java.net.URL
+import java.net.http.HttpClient
+import java.net.http.HttpHeaders
+import java.net.http.HttpRequest
+import java.net.http.HttpResponse
+import java.nio.charset.Charset
+import java.nio.charset.StandardCharsets
+
+class EventSource : EventTarget {
+
+    var bffr: StringBuffer = StringBuffer()
```

Comment:
A string buffer should be encapsulated ( so at least private ) and ideally inside a function. Re-using string buffers is a bad idea, as it keeps state ( e.g. the buffer ) leading to unpredictable results. It may also lead to issues when multiple threads are accessing the same buffer. Its probably not thread-safe.